ddln_imaginator.proto

```diff
@@ -21,6 +21,64 @@ enum ImageFormat {
     GRAY = 4;
 }
 
+message Environment {
+    message Clouds {
+        enum Type {
+            NONE = 0;
+            ALTOCUMULUS = 1;
+            CLOUDS_BASE = 2;
+            CUMULUS = 3;
+            NIMBOSTRATUS = 4;
+            STRATOCUMULUS = 5;
+            STRATUS = 6;
+        }
+        optional Type type = 1;
+        optional float altitude_m = 2;
+        optional Point3d velocity = 3; // Only use the directional x,y,z values to set the wind in the Engine.
+    }
+    optional Clouds clouds = 1;
+
+    message Haze {
```

Comment:
you need some comment to explain difference between `Haze` and `Fog`

Reply:
I'll add this: https://drive.google.com/open?id=1OQ6HtqgzmyRXe6VHeSQTdshwb1162kzx

Reply:
A small comment would be preferred. Note: this interfaces are public and not everyone has access to our internal documentation.

Reply:
Haze // Small dry particles in the air. Characteristics: keeps the brightness and color of the light
Fog // Water droplets in the air. Characteristics: white colored fog which obstructs the light source